pages/your_tasks/data_protection.md

```diff
@@ -26,11 +26,11 @@ faircookbook:
 
 ### Description
 
-Where scientific research involves the processing of data concerning people in the EU, it is subject to the General Data Protection Regulation (GDPR). The GDPR applies a ["special regime"](https://edps.europa.eu/sites/edp/files/publication/20-01-06_opinion_research_en.pdf) to research, providing
+Where scientific research involves the processing of data concerning people in the European Union (EU), it is subject to the General Data Protection Regulation (GDPR). The GDPR applies a ["special regime"](https://edps.europa.eu/sites/edp/files/publication/20-01-06_opinion_research_en.pdf) to research, providing
```

Comment:
just wondering if it's a good idea to describe/define this link in text a bit? I thought it would be GDPR document, but the PDF is by EU's EDPS.